[List of Software](../software_shortlist.md)  
# FastANI 1.33  
![Static Badge](https://img.shields.io/badge/class-bio-D7D1CC)  
  
  
FastANI is developed for fast alignment-free computation of
 whole-genome Average Nucleotide Identity (ANI). ANI is defined as mean
 nucleotide identity of orthologous gene pairs shared between two microbial
 genomes. FastANI supports pairwise comparison of both complete and draft
 genome assemblies.  
  
  
## Homepage  
The homepage for FastANI 1.33   
[https://github.com/ParBLiSS/FastANI](https://github.com/ParBLiSS/FastANI)  
### Usage  
To use this module on Bunya, load it with this command:  
  **`module load fastani/1.33-gcc-10.3.0`**  
  
### Toolchain  
GCC-10.3.0  
  
### Toolchain Options  
(none)  
## Available on architecture types  
- epyc3 a100  
- epyc3 h100  
- epyc3 l40  
- epyc3 mi210  
- epyc3  
- epyc4 a16  
- epyc4 l40s  
- epyc4 mi210  
- epyc4  
  
  
## Dependencies  
FastANI depends on the following modules, and they will be loaded automatically with this module.  
  
[GSL 2.7  (GCC-10.3.0)](gsl-2.7-gcc-10.3.0.md)  
[zlib 1.2.11  (GCCcore-10.3.0)](zlib-.1.2.11-gcccore-10.3.0.md)  
  
## Loaded by  
The following modules will load FastANI as dependency.  
  
[GTDB-Tk 2.0.0  (foss-2021a)](gtdb-tk-2.0.0-foss-2021a.md)  
  
