[List of Software](../software_shortlist.md)  
----  
  
# Trinity 2.9.1  
![Static Badge](https://img.shields.io/badge/class-bio-D7D1CC)  
  
  
Trinity represents a novel method for the efficient and robust de novo reconstruction
 of transcriptomes from RNA-Seq data. Trinity combines three independent software modules: Inchworm,
 Chrysalis, and Butterfly, applied sequentially to process large volumes of RNA-Seq reads.  
  
  
## Homepage  
The homepage for Trinity 2.9.1   
[https://trinityrnaseq.github.io](https://trinityrnaseq.github.io)  
### Usage  
To use the module on Bunya, load it with this command:  
  **`module load trinity/2.9.1-foss-2021a`**  
  
### Toolchain  
foss-2021a  
  
### Toolchain Options  
optarch: True  
## Available on architecture types  
- epyc3 a100  
- epyc3 h100  
- epyc3 l40  
- epyc3 mi210  
- epyc3  
- epyc4 a16  
- epyc4 l40s  
- epyc4 mi210  
- epyc4  
  
  
## Dependencies  
Trinity depends on the following modules, and they will be loaded automatically with this module.  
  
[Java 11  (system-system)](java-11.md)  
[ant 1.10.9 -Java-11 (system-system)](ant-1.10.9-java-11.md)  
[Perl 5.32.1  (GCCcore-10.3.0)](perl-5.32.1-gcccore-10.3.0.md)  
[Python 3.9.5  (GCCcore-10.3.0)](python-3.9.5-gcccore-10.3.0.md)  
[SciPy-bundle 2021.05  (foss-2021a)](scipy-bundle-2021.05-foss-2021a.md)  
[Bowtie2 2.4.4  (GCC-10.3.0)](bowtie2-2.4.4-gcc-10.3.0.md)  
[HTSlib 1.12  (GCC-10.3.0)](htslib-1.12-gcc-10.3.0.md)  
[Jellyfish 2.3.0  (GCC-10.3.0)](jellyfish-2.3.0-gcc-10.3.0.md)  
[kallisto 0.48.0  (gompi-2021a)](kallisto-0.48.0-gompi-2021a.md)  
[ncurses 6.2  (GCCcore-10.3.0)](ncurses-.6.2-gcccore-10.3.0.md)  
[Salmon 1.4.0  (gompi-2021a)](salmon-1.4.0-gompi-2021a.md)  
[SAMtools 1.13  (GCC-10.3.0)](samtools-1.13-gcc-10.3.0.md)  
[zlib 1.2.11  (GCCcore-10.3.0)](zlib-.1.2.11-gcccore-10.3.0.md)  
  
## Dependended on  
The following modules will load Trinity as dependency.  
  
  
