# Trinity 2.9.1  
![Static Badge](https://img.shields.io/badge/class-bio-D7D1CC)  
  
  
Trinity represents a novel method for the efficient and robust de novo reconstruction
 of transcriptomes from RNA-Seq data. Trinity combines three independent software modules: Inchworm,
 Chrysalis, and Butterfly, applied sequentially to process large volumes of RNA-Seq reads.  
  
  
### Usage  
To use the module on Bunya, load it with the command  
  **`module load trinity/2.9.1-foss-2021a`**.  
  
### Toolchain  
foss (2021a)  
  
### Toolchain Options  
{'optarch': True}  
  
  
## Available on architecture types  
- epyc3 a100  
- epyc3 h100  
- epyc3 l40  
- epyc3 mi210  
- epyc3  
- epyc4 a16  
- epyc4 l40s  
- epyc4 mi210  
- epyc4  
  
  
## Dependencies  
[Java 11  (system system)](link)  
[ant 1.10.9 -Java-11 (system system)](link)  
[Perl 5.32.1  (GCCcore 10.3.0)](link)  
[Python 3.9.5  (GCCcore 10.3.0)](link)  
[SciPy-bundle 2021.05  (foss 2021a)](link)  
[Bowtie2 2.4.4  (GCC 10.3.0)](link)  
[HTSlib 1.12  (GCC 10.3.0)](link)  
[Jellyfish 2.3.0  (GCC 10.3.0)](link)  
[kallisto 0.48.0  (gompi 2021a)](link)  
[ncurses 6.2  (GCCcore 10.3.0)](link)  
[Salmon 1.4.0  (gompi 2021a)](link)  
[SAMtools 1.13  (GCC 10.3.0)](link)  
[zlib 1.2.11  (GCCcore 10.3.0)](link)  
  
## Homepage  
The homepage for Trinity 2.9.1 .  
[https://trinityrnaseq.github.io](https://trinityrnaseq.github.io)  
