[< Software](../software_shortlist.md)  
# minimap2  
Category: ['gffcompare/0.12.2-gcc-10.3.0', '/sw/auto/rocky8d/xeonsp4/modules/all/gffcompare/0.12.2-gcc-10.3.0']  
## Homepage  
[https://github.com/lh3/minimap2](https://github.com/lh3/minimap2)  
## Installed versions  
  
| Version | Toolchain | Options | Module |  
| :--------- | :--- | :--- |  
 | [minimap2 2.24](../modules/minimap2-2.24-gcccore-11.3.0.md) | GCCcore-11.3.0 |  | minimap2/2.24-gcccore-11.3.0 |  
  
Minimap2 is a fast sequence mapping and alignment
program that can find overlaps between long noisy reads, or map long
reads or their assemblies to a reference genome optionally with detailed
alignment (i.e. CIGAR). At present, it works efficiently with query
sequences from a few kilobases to ~100 megabases in length at an error
rate ~15%. Minimap2 outputs in the PAF or the SAM format. On limited
test data sets, minimap2 is over 20 times faster than most other
long-read aligners. It will replace BWA-MEM for long reads and contig
alignment.  
  
----  
  
