[List of Software](../software_shortlist.md)  
# Trinity  
![Static Badge](https://img.shields.io/badge/class-bio-D7D1CC)  
## Homepage  
[https://trinityrnaseq.github.io](https://trinityrnaseq.github.io)  
## Installed versions  
  
| Version | Toolchain | Module | Architecture 
| :--------- | :--- | :--- | ---: |  
 | Trinity 2.9.1 | foss-2021a | [trinity/2.9.1-foss-2021a](../modules/trinity-2.9.1-foss-2021a.md) | ![Static Badge](https://img.shields.io/badge/epyc3--A100-ADA9A6) ![Static Badge](https://img.shields.io/badge/epyc3--H100-4085C6) ![Static Badge](https://img.shields.io/badge/epyc3--L40-BB9D65) ![Static Badge](https://img.shields.io/badge/epyc3--mi210-FBB800) ![Static Badge](https://img.shields.io/badge/epyc3-D7D1CC) ![Static Badge](https://img.shields.io/badge/epyc4--A16-999490) ![Static Badge](https://img.shields.io/badge/epyc4--L40s-2EA836) ![Static Badge](https://img.shields.io/badge/epyc4--mi210%2Fepyc4--mi300x-EB602B) ![Static Badge](https://img.shields.io/badge/epyc4-D6D4D3)  |  
  
Trinity represents a novel method for the efficient and robust de novo reconstruction
 of transcriptomes from RNA-Seq data. Trinity combines three independent software modules: Inchworm,
 Chrysalis, and Butterfly, applied sequentially to process large volumes of RNA-Seq reads.  
  
----  
  
